extensions/ARB/ARB_draw_elements_base_vertex.txt

```diff
@@ -128,8 +128,8 @@ New Procedures and Functions
     void DrawElementsInstancedBaseVertex(enum mode, sizei count,
          enum type, const void *indices, sizei primcount, int basevertex);
 
-    void MultiDrawElementsBaseVertex(enum mode, sizei *count, enum type,
-         void **indices, sizei primcount, int *basevertex)
+    void MultiDrawElementsBaseVertex(enum mode, const sizei *count, enum type,
+         const void *const *indices, sizei primcount, const int *basevertex)
```

Comment:
Might as well fix the name of the second last parameter as well?  (spec and headers now call it "drawcount" instead of "primcount").

Reply:
Sure, why not.